R/local_storage.R

```diff
@@ -0,0 +1,92 @@
+
+#' @title Local Storage
+#' 
+#' @description
+#' Class for accessing the browser local storage from within a *Shiny* app.
+#' 
+LocalStorage <- R6::R6Class(
+  classname = "Local Storage",
+  public = list(
+    #' @description
+    #' Create a new instance of this [R6][R6::R6Class] class.
+    initialize = function() {
+      # Get root session so that storage is not tied to module namespace
+      private$session <- shiny::getDefaultReactiveDomain()$rootScope()
+      private$storage <- reactive({
```

Comment:
You've scoped the `getDefaultReactiveDomain`, but not this `reactive`, might need some consistency.

Reply:
Fixed